R/colors.R

```diff
@@ -30,6 +30,15 @@
 #' @md
 #' @export
 parseCssColors <- function(str, mustWork = TRUE) {
+  # Logic below assumes a character string with non-missing values
+  # Note that an empty string is not a valid color, so parsing fails
+  # on NA input values, and thus, will be converted back to NA
+  # when `mustWork = FALSE`
+  if (!(is.character(str) || rlang::is_na(str))) {
```

Comment:
But what about the cases where it's a vector of (non-character) `NA`s? As in:

```R
parseCssColors(c(NA, NA))
```

Reply:
For some reason I was thinking that'd be a character vector, but I suppose logical makes way more sense!